pattern/windows/windows.xml

```diff
@@ -0,0 +1,173 @@
+ Logon Process:  User32 
+ Authentication Package: Negotiate
+ Transited Services: -
+ Package Name (NTLM only): -
+ Key Length:  0
+
+This event is generated when a logon session is created. It is generated on the computer that was accessed.
+
+The subject fields indicate the account on the local system which requested the logon. This is most commonly a service such as the Server service, or a local process such as Winlogon.exe or Services.exe.
+
+The logon type field indicates the kind of logon that occurred. The most common types are 2 (interactive) and 3 (network).
+
+The New Logon fields indicate the account for whom the new logon was created, i.e. the account that was logged on.
+
+The network fields indicate where a remote logon request originated. Workstation name is not always available and may be left blank in some cases.
+
+The authentication information fields provide detailed information about this specific logon request.
+ - Logon GUID is a unique identifier that can be used to correlate this event with a KDC event.
+ - Transited services indicate which intermediate services have participated in this logon request.
+ - Package name indicates which sub-protocol was used among the NTLM protocols.
+ - Key length indicates the length of the generated session key. This will be 0 if no session key was requested. (EventID 4624)</test_message>
+                         <test_values>
+                             <test_value name='win.EventID'>4624</test_value>
+                             <test_value name='win.TargetUserName'>Administrator</test_value>
+                             <test_value name='win.SubjectDomainName'>BSP</test_value>
+                             <test_value name='win.IpAddress'>10.20.30.40</test_value>
+                             <test_value name='win.IpPort'>56789</test_value>
+                             <test_value name='win.WorkstationName'>BSP-WIN2K8R2-EN</test_value>
+                             <test_value name='win.TargetLogonId'>0xdeadbeef</test_value>
+                             <test_value name='win.ProcessName'>C:\Windows\System32\winlogon.exe</test_value>
+                             <test_value name='win.ProcessId'>0xabc</test_value>
+                             <test_value name='win.LogonType'>10</test_value>
+                             <test_value name='win.TargetDomainName'>BSP-WIN2K8R2-EN</test_value>
+                             <test_value name='win.LogonGuid'>{00000000-0000-0000-0000-000000000000}</test_value>
+                             <test_value name='win.AuthenticationPackageName'>Negotiate</test_value>
+                             <test_value name='win.LogonProcessName'>User32 </test_value>
+                             <test_value name='win.LmPackageName'>-</test_value>
+                             <test_value name='win.TransmittedServices'>-</test_value>
+                             <test_value name='win.KeyLength'>0</test_value>
+                         </test_values>
```

Comment:
name value pairs should start with "." to indicate a reserved key-value namespace.